diff --git a/dmenu_dex_run b/dmenu_dex_run
index 364ad4d..6c861b8 100755
--- a/dmenu_dex_run
+++ b/dmenu_dex_run
@@ -5,8 +5,8 @@ selected=$(echo "$apps" | cut -d "$(printf "\t")" -f1 | \
   dmenu -i -F -l 20 -z 480 -p "Applications")
 
 selected="$(echo "$apps" | grep "^$selected$(printf "\t")")"
-[ -n "$selected" ] && \
-  setsid -f xdg-open "$(echo "$selected" | cut -d "$(printf "\t")" -f2)"
+[ -n "$selected" ] && echo "$selected" | cut -d "$(printf "\t")" -f2 | \
+	xargs basename | xargs setsid -f gtk-launch
 
 unset apps selected
 exit 0
